sci-biology/hmmer/hmmer-3.1_beta2.ebuild

```diff
@@ -9,7 +9,7 @@ inherit eutils
 MY_PV="3.1b2"
```

Comment:
I don't modify this part of code, it is like this in master branch.
versionator eclass is smart enough to translate the gentoo version
3.1_beta2 in url version 3.1b2?
or have i to change the version of the ebuild for 3.1b2?

thank's

bertrand


On Sat, Jul 23, 2016 at 1:10 PM, Marius Brehler <notifications@github.com>
wrote:

> In sci-biology/hmmer/hmmer-3.1_beta2.ebuild
> <https://github.com/gentoo-science/sci/pull/651#discussion_r71971146>:
>
> > @@ -9,7 +9,7 @@ inherit eutils
> >  MY_PV="3.1b2"
>
> The version shoudn't be hardcoded. Give the versionator eclass a try
> instead.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/gentoo-science/sci/pull/651/files/dfa3f177ba681a327737f1b1d0805f78071441a9#r71971146>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ALDS5zOmy_HJiB_VJE5_bNA60s6AYOMyks5qYfatgaJpZM4JS5xq>
> .
>